src/udev/scsi_id/scsi_id.c

```diff
@@ -59,6 +61,8 @@ static char model_enc_str[256];
 static char revision_str[16];
 static char type_str[16];
 
+enum scsi_id_interface scsi_interface = SCSI_ID_INTERFACE_UNKNOWN;
```

Comment:
Please prefix the global variable that is controlled by the command line option with `arg_`.

Reply:
Done